src/Upgrader/Business/Composer/ComposerJson/ComposerJsonReader.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Copyright © 2021-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Business\Composer\ComposerJson;
+
+use Upgrader\Business\Composer\Helper\JsonFile\JsonFileReadHelper;
+
+class ComposerJsonReader implements ComposerJsonReaderInterface
+{
+    protected const FILENAME_JSON = 'composer.json';
+
+    /**
+     * @var \Upgrader\Business\Composer\Helper\JsonFile\JsonFileReadHelper
+     */
+    protected $jsonFileReadHelper;
+
+    /**
+     * @param \Upgrader\Business\Composer\Helper\JsonFile\JsonFileReadHelper $jsonFileReadHelper
+     */
+    public function __construct(JsonFileReadHelper $jsonFileReadHelper)
+    {
+        $this->jsonFileReadHelper = $jsonFileReadHelper;
+    }
+
+    /**
+     * @return array
+     */
+    public function read(): array
+    {
+        return $this->jsonFileReadHelper->readFromPath(self::FILENAME_JSON);
```

Comment:
As for me helper is not required as it's just a proxy.
Please move the logic from Helper to the current Reader and the same think for Composer writer.